Converter.php

```diff
@@ -280,13 +303,13 @@ protected function _composeStrWithIndents($lines, $indentSize) {
 	}
 
 	/**
-	 * Uses tpl file for output result.
-	 * 
+	 * Use template file for output result.
+	 *
 	 * @param string $tplFile filepath
 	 * @param string $testMethodContent
 	 * @return string output content
 	 */
-	protected function _convertToTpl($tplFile, $testMethodContent = null) {
+	protected function _convertToTpl(string $tplFile, string $testMethodContent = null) {
```

Comment:
Hmm - så bør du angive minimum requirement i composer.json til at være PHP7+. Ved ikke om det er et requirement vi gerne vil have... men på den anden side så "hvorfor ikke?"

Reply:
Det mente jeg at have gjort ... men nej, det var for QA-projektet :-/